internal/generate/clusterserviceversion/clusterserviceversion_updaters.go

```diff
@@ -316,16 +336,80 @@ func mutatingToWebhookDescription(webhook admissionregv1.MutatingWebhook) operat
 		AdmissionReviewVersions: webhook.AdmissionReviewVersions,
 		ReinvocationPolicy:      webhook.ReinvocationPolicy,
 	}
+	if len(description.AdmissionReviewVersions) == 0 {
+		description.AdmissionReviewVersions = defaultAdmissionReviewVersions
+	}
+
```

Comment:
The same `sideEffects` and `containerPort` as in validating WH needs to be set here too.